src/worker/start.rs

```diff
@@ -198,6 +199,15 @@ async fn launcher_main(streamer_ref: StreamerRef, task_ref: TaskRef) -> tako::Re
             .insert(HQ_INSTANCE_ID.into(), task.instance_id.to_string().into());
 
         replace_placeholders(&mut program);
+        // Creates all not existing folders given in stdout
+        let std_folders = match &program.stdout {
+            StdioDef::File(path) => match path.parent() {
+                Some(path) => path.to_str().unwrap(),
+                None => "",
+            },
+            _ => "",
+        };
+        std::fs::create_dir_all(std_folders)?;
```

Comment:
You can pass `PathBuf` to `create_dir_all` directly. Try something like
```rust
if let StdioDef::File(path) = program.stdout {
    std::fs::create_dir_all(path)?;
}
```
Please also do the same for `stderr`.